sslutils--1.4.sql

```diff
@@ -26,24 +26,26 @@ COMMENT ON FUNCTION openssl_rsa_key_to_csr(text, text, text, text, text, text, t
 -- param6 : OU or Organization Unit
 -- param7 : email.';
 
-CREATE OR REPLACE FUNCTION openssl_csr_to_crt(text, text, text)
+CREATE OR REPLACE FUNCTION openssl_csr_to_crt(text, text, text, integer DEFAULT NULL)
 RETURNS text
 AS 'MODULE_PATHNAME', 'openssl_csr_to_crt'
 LANGUAGE C IMMUTABLE;
-COMMENT ON FUNCTION openssl_csr_to_crt(text, text, text) IS 'Generates the CA/self signed certificate.
+COMMENT ON FUNCTION openssl_csr_to_crt(text, text, text, integer) IS 'Generates the CA/self signed certificate.
 -- Parameters:
 -- param1 : csr or certificate signing request
 -- param2 : Path to CA certificate OR NULL If CA self signed certificate is required.
--- param3 : Path to CA private key OR Path to self private key, If param2 is NULL.';
+-- param3 : Path to CA private key OR Path to self private key, If param2 is NULL.
+-- param4 : The validity days of generated certificate.';
```

Comment:
s/The validity days of generated certificate/The validity (in days) for the generated certificate